### ISSUE TITLE
List and Alternatives of open-source searching (multi-engines) addons

### ISSUE THREAD

**Issue by garywill** [CanisLupus/swift-selection-search#236]
Hello,  Swift Selection Search developers and users ~

I see the NOTE that SSS addon is no longer being maintained. 
So please allow me to post a list and comparison of similar addons:

![compare-search](https://user-images.githubusercontent.com/32130780/142715271-5f2d7f97-5fac-450a-9f83-5b9e4dc8c7c3.png)
You can find the entire list [here in my repo](https://github.com/garywill/BigSearch/blob/list/list.md)

Actually [I'm the developer of one such an addon](https://github.com/garywill/BigSearch). I invite you guys to try my addon too ~

**Comment by CanisLupus** [comment]
Hi Gary! I'm the main developer for Swift Selection Search. I forgot to reply when you first posted this. 😅

Although I haven't used Big Search, I still want to congratulate and encourage you on your endeavor to develop it! It's nice to have other options.

All the best to you and happy new year, whether you celebrate it now or later! 🎉

**Comment by garywill** [comment]
@CanisLupus Thank you, sir! You truely did an awesome work on SSS. 
Also thank you for your encouragement  :) 
Happy new year to you too! :tada:

**Comment by garywill** [comment]
> The developer of BigSearch is Chinese, which could explain why the sites being used are Chinese companies, but I'm not happy with sending my data to a company that is required by law to report all that data to the Chinese Communist Party.

Hi there @cheater . Here I am. I'm the dev of Big Search

I understand your doubt. Big Search is currently using analystic server just to see how many people are using it
- Sensitive data is **NEVER** collected
- Users **can** turn analystic off completely

(as said in the Privacy Policy on AMO. Also source code is available to read.)

I am **not** doing anything commercial, nor cooporating with any commercial companies.

**Comment by cheater** [comment]
Of course. My worry comes from the reported fact that tech companies located in China are required by law to provide their data to the Chinese government warrant-free. I can't inspect the code well enough to find out what is being and isn't being sent and when it is turned off.

**Comment by garywill** [comment]
> I can't inspect the code well enough to find out what is being and isn't being sent and when it is turned off.

Nothing is sent when user disables. Check this code
https://github.com/garywill/BigSearch/blob/92b79daaa59f238c2099c43ebc483c5993712ece/src/i_scripts.js#L1214-L1219
People can see how the analystics work

**Comment by cheater** [comment]
... why do you even have analytics like that?

**Comment by katastrophe92** [comment]
@cheater Sadly we have lost the fight of being private. It is too hard and uncomfortable to use the interned being somehow private

**Comment by cheater** [comment]
yeah no answer = easy to assume you're being monetized

sorry gary, that's just kinda bs

**Comment by garywill** [comment]
BigSearch has got enough non-sensitive analystics data. Thank those users who provided their non-sensitive data to help this product.
Now analystics is **completely removed**. I don't need them any more.
 See https://github.com/garywill/BigSearch/discussions/8
